interfaces/cython/cantera/test/test_utils.py

```diff
@@ -6,6 +6,20 @@
 from cantera._cantera import _py_to_any_to_py
 
 
+class TestWarnings(utilities.CanteraTest):
+
+    def test_user(self):
+        yaml = """
+            type: falloff
+            low-P-rate-constant: {A: 2.3e+12, b: -0.9, Ea: -7112800.0}
+            high-P-rate-constant: {A: 7.4e+10, b: -0.37, Ea: 0.0}
+            Troe: {A: 0.7346, T3: 94.0, T1: 1756.0, T2: 0.}
+            """
+
+        with self.assertWarnsRegex(Warning, "Unexpected parameter value T2=0"):
```

Comment:
Might as well use the pytest version, `pytest.warns`. Should this be `UserWarning`?

Reply:
This is arguably a `SyntaxWarning` (which are possible with this PR), but I'd prefer to leave all C++ warnings as `CanteraWarnings`, which are now converted to `UserWarnings` on the Python side.